tests/qos/qos_sai_base.py

```diff
@@ -881,6 +881,31 @@ def disablePacketAging(
             duthost.command("docker exec syncd python /packets_aging.py enable")
             duthost.command("docker exec syncd rm -rf /packets_aging.py")
 
+    def dutBufferConfig(self, duthost):
+        bufferConfig = {}
+        try:
+            bufferConfig['BUFFER_POOL'] = json.loads(duthost.shell('sonic-cfggen -d --var-json "BUFFER_POOL"')['stdout'])
+            bufferConfig['BUFFER_PROFILE'] = json.loads(duthost.shell('sonic-cfggen -d --var-json "BUFFER_PROFILE"')['stdout'])
+            bufferConfig['BUFFER_QUEUE'] = json.loads(duthost.shell('sonic-cfggen -d --var-json "BUFFER_QUEUE"')['stdout'])
+            bufferConfig['BUFFER_PG'] = json.loads(duthost.shell('sonic-cfggen -d --var-json "BUFFER_PG"')['stdout'])
+        except Exception as err:
+            logger.info(err)
+        return bufferConfig
+
+    def dutAsicConfig(self, asic, duthost):
+        asicConfig = {}
+        # Only support to get brcm asic info, so far
+        if 'broadcom' in asic.lower():
+            output = duthost.shell('bcmcmd "g THDI_BUFFER_CELL_LIMIT_SP"', module_ignore_errors=True)
```

Comment:
Is this register available for all BRCM asics?

Reply:
on th/th2/td2/td3 platform, command 'bcmcmd "g THDI_BUFFER_CELL_LIMIT_SP"' is always woring.
And add "try except" for THDI_BUFFER_CELL_LIMIT_SP register is unavailable case.